/*package org.example.model;
import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.fail;

public class OrdreDesJoueursTest {

    @Test
    public final void testConstructeur() {
        try {
            OrdreDesJoueurs o = new OrdreDesJoueurs("123", 3);
        } catch (OrdreDesJoueursException oe) {
            fail("OrdreDesJoueurs exception");
        }

    }

}*/


//QUEST CE QUE CA FOUT LA ?????????
//ON A UN DOSSIER TEST NON ?
